codegen/core/src/main/scala/com/lightbend/akkasls/codegen/File.scala

```diff
@@ -39,6 +40,18 @@ case class GeneratedFiles(
     unmanagedTestFiles.map(_.writeToDirectory(unmanagedTestSourceDirectory, onlyIfMissing = true)) ++
     integrationTestFiles.map(_.writeToDirectory(unmanagedIntegrationTestSourceDirectory, onlyIfMissing = true))
 
+  def overwrite(
+      managedSourceDirectory: Path,
+      unmanagedSourceDirectory: Path,
+      managedTestSourceDirectory: Path,
+      unmanagedTestSourceDirectory: Path,
+      unmanagedIntegrationTestSourceDirectory: Path): Iterable[Path] =
+    managedFiles.map(_.writeToDirectory(managedSourceDirectory, onlyIfMissing = false)) ++
+    unmanagedFiles.map(_.writeToDirectory(unmanagedSourceDirectory, onlyIfMissing = false)) ++
+    managedTestFiles.map(_.writeToDirectory(managedTestSourceDirectory, onlyIfMissing = false)) ++
+    unmanagedTestFiles.map(_.writeToDirectory(unmanagedTestSourceDirectory, onlyIfMissing = false)) ++
+    integrationTestFiles.map(_.writeToDirectory(unmanagedIntegrationTestSourceDirectory, onlyIfMissing = false))
```

Comment:
It turns out that we were not regenerating the `unmanaged` files. See above in method `write` (GitHub collapsed). 

This method is only used for testing though. We could eventually move it or make it `private [akkaserverless]`? In any case, codegen is not expected to be used directly by users so no big deal IMO.

Reply:
We should probably mark that whole class as internal API. We never know.

Reply:
We can do it in follow-up PR. I think all the codegen API should be for internal usage only. 

They don't show up as a dependency in the project. In principle they are a dependency for the plugins. Still, they show up in maven central as an artifact. 

At that point in time (and probably later as well), we want to have full freedom to change the codegen code the way we find necessary without having to keep any sort of backward compatibility.